agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/GeneRepository.java

```diff
@@ -642,7 +642,7 @@ private String getAllGenesQuery(OrthologyFilter filter) {
     }
 
     public List<AffectedGenomicModel> getAllAffectedModelsSTR() {
-        String query = " MATCH p=(:AffectedGenomicModel)-[:SEQUENCE_TARGETING_REAGENT]-(:SequenceTargetingReagent)--(gene:Gene)";
+        String query = " MATCH p=(:AffectedGenomicModel)-[:SEQUENCE_TARGETING_REAGENT]-(:SequenceTargetingReagent)--(gene:Gene)-[:FROM_SPECIES]->(:Species)";
```

Comment:
Is there Gene's that don't have a species? or are you just trying to pickup the species node?